core/src/services/oss/backend.rs

```diff
@@ -123,6 +123,7 @@ pub struct OssBuilder {
     access_key_secret: Option<String>,
 
     http_client: Option<HttpClient>,
+    write_buffer_size: Option<usize>,
```

Comment:
Please call this value `write_min_size`.

Reply:
I just noticed that the oss multipart size should be 100 KB to 5 GB, and maybe we should also limit upper bound, although this upper bound is large, it is rarely exceeded.
So I just keep naming it `writer_buffer_size` and do the size checking.

Reply:
No, they are different thing. We will have `write_max_size` in the future.